src/org/omegat/core/dictionaries/DictionaryEntry.java

```diff
@@ -31,14 +32,26 @@
  * @author Alex Buloichik (alex73mail@gmail.com)
  */
 public class DictionaryEntry implements Comparable<DictionaryEntry> {
+    private final String query;
     private final String word;
     private final String article;
 
     public DictionaryEntry(final String word, final String article) {
+        this.query = word.toLowerCase();
```

Comment:
I would have this constructor call the other constructor rather than (mostly) duplicate its body.

Reply:
resolved.